hw/ip/aes/data/aes.hjson

```diff
@@ -194,11 +209,78 @@
       '''
     }
   ],
+
+  /////////////////////
+  // Countermeasures //
+  /////////////////////
+
   countermeasures: [
     { name: "BUS.INTEGRITY",
       desc: "End-to-end bus integrity scheme."
     }
+    { name: "CONFIG.SHADOW",
+      desc: "Main control register shadowed."
+    }
+    { name: "CONFIG.SPARSE",
+      desc: "Critical fields in main control register one-hot encoded."
+    }
+    { name: "DATA_REG.SW_UNREADABLE",
```

Comment:
I wasn't sure whether to put `KEY` or `DATA_REG` here. `DATA_REG` stands for "non-software config data registers (e.g. keccak state)". Could also list the two separately. WDYT?

Reply:
I guess maybe I'd split it, but don't have strong feelings.

Reply:
Sounds good, I've now split it up into to items for `KEY` and `DATA_REG`.